README.md

```diff
@@ -82,81 +63,14 @@ Now that your `Docker` services are ready to be used, start the full CMS by runn
 
 You should be able to view the site at [localhost:8070](http://localhost:8070)
 
-Once the CMS is up and running, you can create a superuser account:
-
-    $ make superuser
-
-You can create a basic demo site by running:
-
-    $ make demo-site
-
-Note that if you don't create the demo site and start from a blank CMS, you will get some errors
-requesting you to create some required root pages. So it is easier as a first approach to test the
-CMS with the demo site.
```

Comment:
I think we should either remove this quickstart guide and point to the documentation, or leave these 2 steps because the site after make bootstrap is really broken. Actually we should move this sentence to after creating the demo site: 
> You should be able to view the site at [localhost:8070](http://localhost:8070)